cli/gh-pr.go

```diff
@@ -94,7 +94,7 @@ func (gr githubRepo) PrTests(pri int, filterRegExStr, splitTestsAt string) (*map
 		}
 
 		// todo thread ctx
-		// nolint: noctx
+		//nolint: noctx
```

Comment:
cli/gh-pr.go:97:3: directive `// nolint: noctx` should be written without leading space as `//nolint: noctx` (nolintlint)
```
                // nolint: noctx
                ^
```